hivemind/optim/optimizer.py

```diff
@@ -183,10 +180,10 @@ def __init__(
         delay_grad_averaging: bool = False,
         delay_state_averaging: bool = True,
         average_state_every: int = 1,
-        use_local_updates: bool = False,
         client_mode: bool = None,
         auxiliary: bool = False,
         grad_compression: CompressionBase = NoCompression(),
+        grad_averager: Optional[GradientAveragerFactory] = PowerSGDGradientAverager.get_factory(averager_rank=32),
```

Comment:
Optional: i would recommend renaming it to grad_averager_cls to match [other similar situations](https://github.com/learning-at-home/hivemind/blob/master/hivemind/moe/server/server.py#L101)